conf/environment.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
   - yosys
   - yosys-plugins
   - vtr=8.0.0.rc2_3575_g253f75b6d
+  - nextpnr-xilinx
   - pip
   - pip:                            # Packages installed from PyPI
     - -r file:requirements.txt
```

Comment:
We probably want an `conf/environment.yml` and a `conf/environment.pinned.yml` file which has specific versions of yosys / vtr / nextpnr / etc. 

Can you log an issue about that?